cmd/openshift-install/migrate/azure/eligible.go

```diff
@@ -0,0 +1,23 @@
+package azure
+
+import (
+	"github.com/spf13/cobra"
+
+	azmigrate "github.com/openshift/installer/pkg/migrate/azure"
+)
+
+func runMigrateAzurePrivateDNSEligibleCmd(cmd *cobra.Command, args []string) error {
+	return azmigrate.Eligible()
+}
+
+// NewMigrateAzurePrivateDNSEligibleCmd adds the eligble command to openshift-install
+func NewMigrateAzurePrivateDNSEligibleCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "eligible",
```

Comment:
so this currently is `openshift-install migrate azure privatedns eligible` ?

Reply:
Now it is azure-privatedns-migrate and azure-privatedns-eligible, which I am not crazy about either, but simpler.